mapsforge-map/src/main/java/org/mapsforge/map/datastore/MapDataStore.java

```diff
@@ -322,7 +322,7 @@ public MapDataStore setPriority(int priority) {
      * @return true if the way should be included in the result set
      */
     public boolean wayAsLabelTagFilter(List<Tag> tags) {
-        return false;
+        return !tags.isEmpty();
```

Comment:
@Sublimis This seems to change the default behavior for all implementations.

This hook is intended to be overridden by the required implementations only, if needed (details [here](https://github.com/mapsforge/mapsforge/issues/776#issuecomment-182261899)).